blog/_posts/2017-09-09-team_code_hoppers_blog.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Our journey - with rgsoc and Open Source!
+layout: post
+created_at: Saturday Sep 09 2017
+permalink: blog/2017-09-09-team_code_hoppers_blog
+current: blog
+author: roselyne
+twitter: code_Hoppers
+categories:
+- student-posts
+---
+
+Being at #rgsoc has been one of the most fruitful yet challenging experiences this year. Every single day was exciting and came with it’s own set of learning curve.
+
+#### Our accomplishments so far?
+
+In a nutshell, we learnt about coding, teamwork, collaboration and issue resolution. We even got to learn how to effectively communicate with remote teams across the world, listening and focusing on a common goal, defining tasks and keeping schedules. Well, sometimes not everything went as we had planned and from that we learnt that nothing is exactly perfect and we always have to keep striving to efficiently communicate and resolve our blockers. **It has been an amazing journey!**
```

Comment:
Sorry for not noticing 2 more small issues. Change 'not everything went' -> 'not everything went **well'**.